web/public/js/directives/dojo/manage-users/template.dust

```diff
@@ -0,0 +1,18 @@
+<div class="dojo-users" class="cd-manage-dojo-users">
+  <h1>{@i18n key="Manage Dojo Users"/}</h1>
+  <uib-tabset active="activeState" justified="true">
+    <uib-tab index="1" ui-sref="manage-dojo-active-users">
+      <uib-tab-heading>
+        <h2 class="h4">{@i18n key="Active"/}</h2>
+      </uib-tab-heading>
+    </uib-tab>
+    <uib-tab index="2" ui-sref="manage-dojo-pending-users">
+      <uib-tab-heading>
+        <h2 class="h4">{@i18n key="Pending"/}</h2>
```

Comment:
"Manage user requests"

We can change this to pending when we include invites as well.